### PR TITLE
Add bug report Q&A to the FAQ

### LINKED ISSUE #4144: There's no information regarding how (or if) players can submit bug reports
Noticed this when I actually had a bug (I think) that I needed to report, but didn't actually know how or where to submit bug reports. Even if there *is* somewhere in the wiki where it's described, it's not in any obvious place (like on the FAQ, Help Centre, or How you can help).

So I propose that there be some section in an article (preferably FAQ) that details how/where to report bugs. (And if there isn't a real way to report bugs, then we should probably explain that.)

### PR DESCRIPTION
**Pull request by paper-toaster**
Resolves #4144

### CODE REVIEW

File: wiki/FAQ/en.md [hunk 1]
```diff
@@ -393,6 +393,10 @@ For any questions regarding pp, please refer to the [wiki page for pp first](/wi
 
 ### General
 
+#### I think I've found a bug! Where can I report it?
+
+If you have a general technical issue, or if you're unsure whether or not the issue you've encountered is a bug or not, you can post your problems on the [Help forums](https://osu.ppy.sh/community/forums/5). If you think you've found a specific, critical issue with osu!, you can create an issue on the [osu-stable GitHub page](https://github.com/ppy/osu-stable-issues).
```
Comment (Trigonoculus) [inline wiki/FAQ/en.md:398]:
```suggestion
If you have a general technical issue, or if you're unsure whether or not the issue you've encountered is a bug or not, you can post your problems on the [Help forums](https://osu.ppy.sh/community/forums/5). If you think you've found a specific, critical issue with osu!, you can create an issue on the [osu-stable-issues GitHub repository](https://github.com/ppy/osu-stable-issues).
```
Suggestion, this wording is also present [here](https://osu.ppy.sh/community/forums/topics/338103).

File: wiki/FAQ/en.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ If [Auto Pilot](/wiki/Game_modifier/Autopilot) game modifier was used, only the
...
 
-Click on the `Mode` button to **open up a list of gameplay modes available on osu!**. Click on the desired gameplay mode, and osu! will switch to that gameplay mode style, along with relevant leaderboard change. Alternatively, press `Ctrl` key and `1` (![][osu! icon] *[osu!standard](/wiki/Game_mode/osu!))* / `2` ( ![][osu!taiko icon] *[osu!taiko](/wiki/Game_mode/osu!taiko)*) / `3` (![][osu!catch icon] *[osu!catch](/wiki/Game_mode/osu!catch)*) / `4` (![][osu!mania icon] *[osu!mania](/wiki/Game_mode/osu!mania)*) key on the keyboard to quickly switch to the desired gameplay mode.
+Click on the `Mode` button to **open up a list of gameplay modes available on osu!**. Click on the desired gameplay mode, and osu! will switch to that gameplay mode style, along with relevant leaderboard change. Alternatively, press `Ctrl` key and `1` (![][osu! icon] *[osu!standard](/wiki/Game_mode/osu!))* / `2` ( ![][osu!taiko icon] *[osu!taiko](/wiki/Game_mode/osu!taiko)*) / `3` (![][osu!catch icon]*[osu!catch](/wiki/Game_mode/osu!catch)*) / `4` (![][osu!mania icon] *[osu!mania](/wiki/Game_mode/osu!mania)*) key on the keyboard to quickly switch to the desired gameplay mode.
```
Comment (Trigonoculus) [inline wiki/FAQ/en.md:103]: Was this unintentional?

**Review by Trigonoculus — CHANGES_REQUESTED** [review]

File: wiki/FAQ/en.md (hunk 1 above)
Comment (TPGPL) [inline wiki/FAQ/en.md:398]:
```suggestion
If you have a general technical issue, or if you're unsure whether the issue you've encountered is a bug or not, you can create a post in the [Help forum](https://osu.ppy.sh/community/forums/5). If you think you've found a specific, critical issue with osu!, you can create an issue in the [osu-stable-issues GitHub repository](https://github.com/ppy/osu-stable-issues/issues).
```

File: wiki/FAQ/en.md (hunk 2 above)
Comment (TPGPL) [inline wiki/FAQ/en.md:103]:
```suggestion
Click on the `Mode` button to **open up a list of gameplay modes available on osu!**. Click on the desired gameplay mode, and osu! will switch to that gameplay mode style, along with relevant leaderboard change. Alternatively, press `Ctrl` key and `1` (![][osu! icon] *[osu!standard](/wiki/Game_mode/osu!))* / `2` ( ![][osu!taiko icon] *[osu!taiko](/wiki/Game_mode/osu!taiko)*) / `3` (![][osu!catch icon]* [osu!catch](/wiki/Game_mode/osu!catch)*) / `4` (![][osu!mania icon] *[osu!mania](/wiki/Game_mode/osu!mania)*) key on the keyboard to quickly switch to the desired gameplay mode.
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/FAQ/en.md (hunk 2 above)
Comment (paper-toaster) [reply wiki/FAQ/en.md:103]: Most likely

File: wiki/FAQ/en.md (hunk 1 above)
Comment (paper-toaster) [reply wiki/FAQ/en.md:398]: Yeah, but I doubt many people would default going to the forums street gimme finding a big.

**Review by TPGPL — APPROVED** [review]

**Pull request merged**